apiproxy/access_api_proxy.go

```diff
@@ -1,468 +0,0 @@
-package apiproxy
```

Comment:
I think it works but the handler wasn't set properly in the observer builder so I decided to simplify things. I will add back the round robin / multiple ANs in the PR for metrics and logs. I'm planning on writing something that will include metrics/logging but also let us configure per RPC how to handle it. I'm thinking: 
- forward immediately
- handle locally if there's an error then forward
- only handle locally

Reply:
for the current observer grpc, you can keep the forward/local logic the same as what was there. There are certain endpoints that should always be served locally, and others that must be forwarded. I think if we're going to add smarts and better failover logic, that should probably go into the API service.

since the existing code went through several rounds of code review already, I think we should fix what's not working instead of a complete rewrite.